ps_wirepayment.php

```diff
@@ -92,7 +92,8 @@ public function __construct()
     public function install()
     {
         Configuration::updateValue(self::FLAG_DISPLAY_PAYMENT_INVITE, true);
-        if (!parent::install() || !$this->registerHook('paymentReturn') || !$this->registerHook('paymentOptions')) {
+        if (!parent::install() || !$this->registerHook('paymentReturn') || !$this->registerHook('paymentOptions') ||
+        !$this->registerHook('actionGetExtraMailTemplateVars')) {
```

Comment:
Don't you need to register new hook in `upgrade` file as well? :)

Btw, it would be nice if you could format this `if` into multiple lines for better readability.

Reply:
I'm not sure in myself )))) doing an upgrade needs a new version for this module. But sure you're right. There should be upgrade.